lib/install.js

```diff
@@ -423,6 +424,44 @@ function install (gyp, argv, callback) {
 
 }
 
+function formatHostname (hostname) {
+  // canonicalize the hostname, so that 'oogle.com' won't match 'google.com'
+  return hostname.replace(/^\.*/, '.').toLowerCase()
```

Comment:
This transforms `google.com` into `.google.com`.  I infer that is intentional but can you update the comment to make that more explicit?  This behavior also seems only lightly tested.

Aside: can you capitalize and punctuate comments?